packages/lab/src/__tests__/__e2e__/list/List.keyboardNavigation.cy.tsx

```diff
@@ -0,0 +1,678 @@
+import { List, ListItem } from "@jpmorganchase/uitk-lab";
+
+const HIGHLIGHTED = "uitkListItem-highlighted";
+const FOCUSVISIBLE = "uitkFocusVisible";
+
+type ItemWithLabel = { label: string };
+const ITEMS: ItemWithLabel[] = [
+  { label: "list item 1" },
+  { label: "list item 2" },
+  { label: "list item 3" },
+  { label: "list item 4" },
+];
+
+const ITEMS_PER_PAGE = 2;
+
+const expectOptionHightlightAt = (index: number) => {
+  cy.get(`#list-item-${index}`).should("have.class", HIGHLIGHTED, FOCUSVISIBLE);
```

Comment:
I've added assertions for `highlighted`, `focusVisible` and `ariaSelected`. There are existing built-in assertions for `selected` and `disabled` which are not useful in this scenario. It would be nice to add our own check for `disabled" to check for the `uitkDisabled` utility class. Clashes with the built-in `disabled` assertion though. Maybe something like `uitk.disabled` ? (in which case we should perhaps go for  `uitk.selected` and `uitk.highlighted` as well ?